lib/tasks/edition.rake

```diff
@@ -0,0 +1,13 @@
+namespace :edition do
+  desc 'Check if any deleted edition has undeleted attachments, and delete them'
+  task delete_edition_files: :environment do
+    Edition.where(state: :deleted).find_each do |edition|
+      clear = false
+      edition.attachments.where(deleted: false).each do |attachment|
+        clear = true
+        puts "deleted edition #{id} has undeleted attachment with url #{attachment.url}"
```

Comment:
Would it be better just to print the legacy ID to make it easier to copy this directly into the Rake task?